plugins/woocommerce-blocks/assets/js/blocks/coming-soon/index.tsx

```diff
@@ -15,6 +15,7 @@ import deprecated from './deprecated';
 import NewsletterPanel from './newsletter-panel';
 import './store-only.scss';
 import './entire-site.scss';
+import './entire-site-deprecated.scss';
```

Comment:
I think we should only enqueue the entire-site-deprecated stylesheet when the deprecated template is in use to avoid the [performance issue](https://github.com/woocommerce/woocommerce/issues/54077)

Reply:
Thanks for catching that, @chihsuan! Unfortunately it wasn't simple to add a conditional CSS asset using block's JS build, so I changed it to use enqueue [frontend](https://github.com/woocommerce/woocommerce/pull/56210/files#diff-45e03efebf7424b6f8b494a3f3941289c6fac876171c7183c8ef63c9770f8911R52-R56) and [site-editor](https://github.com/woocommerce/woocommerce/pull/56210/files#diff-45e03efebf7424b6f8b494a3f3941289c6fac876171c7183c8ef63c9770f8911R64-R94)